### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by weijia-cheng** [Gragt/mary-shelley_the-last-man#1]
# Spelling changes

- [x] `chapter-2-2`: Muhammad -> Mehmed in "Muhammad had made the breach..." This originally said "Mahomet." It seems to me that this is referring to [Mehmed II](https://en.wikipedia.org/wiki/Mehmed_II)'s conquest of Constantinople. Spelling his name as "Muhammad" isn't *wrong*, exactly (it is the Arabic form of his name) but it seems confusing because 1) that's not how his name is usually spelled and 2) "Muhammad" without any regnal numbers or epithets makes me think of the Islamic prophet, which is incorrect in this context.
- [x] Ispahan -> Isfahan
- [x] Cashmere -> Kashmir
- [x] `chapter-3-4`: "sand-bank" was modernized to "sandbank" in verse. I would revert this because "sand-bank" has a different stress pattern than "sandbank."
- [x] 7e76e1586695ed2658365182f3cc63c16f11d94c: I'm not sure I agree with the decision to normalize "Ex-Queen" as "ex-queen." Since "Ex-Queen" seems to be referring to a particular person, I think the original capitalization is fine.
- [x] baea7b3260778898b9dca82e66b1e0f29f382240: Is "Elect" here referring to a particular group of people? I can see the "Ex-Queen" one going either way but I think this one should be reverted because it seems to be the name of a movement.

# Other stuff

- [x] `chapter-3-3`: You used the timestamp `2097-30-12` where it should have been `2097-12-30`.
- [x] 7806e2dab8dacb2d40ba5e4371c3690352670f0a: In this editorial commit, you combined spelling changes with semantication (the span you added around "Laplace"). Unfortunately we've established that this is a no-no in terms of commit hygiene and you should rebase the span addition out of this commit and add it in separately.
- [x] d9e16f92a1ec0ee2882c78f9e8c0d6088b182840: In this commit, you added in quotation marks around thoughts. I suppose this is fine as is, but if we are changing the formatting, we might as well follow [8.2.7](https://standardebooks.org/manual/1.7.0/8-typography#8.2.7) and use `<q>` for the thoughts.
- [x] The `part-*` files/sections should be renamed `volume-*` and use the `volume` epub semantic.
- [x] In the CSS, I think the `#chapter-3-10 blockquote:last-of-type p` rule can be consolidated with the `#chapter-3-10 blockquote:last-of-type` rule for simplicity's sake.
- [x] "Over the years, she had lost her husband, Percy Bysshe Shelley, in a boating accident; her close friend, Lord Byron; and her children." -- I think the wording of this sentence in the long description is confusing. "In a boating accident" coming in the middle of what is otherwise a list of people makes the sentence hard to follow.

**Comment by Gragt** [comment]
I’ve finally had time to effect the changes. I should’ve known it would have to wait until the weekend.

This involved a few rebases, so you’ll need to force the updates on your end to see the final result—as I’m sure you’re already aware of.

A a couple of comments:

By Jove, you’re absolutely right about Mehmet! The toolset made the change and it didn’t occur to me that it couldn’t be the prophet. I’ve made the change by editing the original commit so it would leave Mahomet intact then inserted another editorial commit right after to update it to Mehmed.

Semantics and editorial: I’m aware of the rule and paid special care so it would not happen. Or so I thought—good catch on your part! Both changes are in their relevant commits as originally intended.

Italicising thoughts: I thought 8.2.7 only applied if the original content was italicised, which isn’t the case here. I’ve made the change by splitting the original commit, one for speech and one for thoughts—the former is tagged as editorial;the latter isn’t since it doesn’t modify original content.

Long description: I used semicolons specifically to separate those elements and avoid confusion, but I agree that this detail doesn’t bring much and can be excised—It was meant to tie with the book in which a character modelled after her husband likewise dies in a boating accident.

And a few things for you to possibly reconsider:

Ex-Queen: the original text has it both capitalised and lowercase, being first encountered lowercase—again, the whole thing lacked a good editor! I checked the CMOS (our doesn’t seem to have anything about titles) and the [general rule](https://archive.org/details/chicagomanualofs16edunse/page/392/mode/2up) appears to be that titles aren’t capitalised unless followed by a name or in a direct address. Interestingly enough, the text also refers to the person of the king in lowercase and the only time the word is capitalised is when it refers to the office and not the person. I’ve made the change but in view of this do you think we should keep it? Should I also capitalise ”king”, the person? Or just leave things as they currently are?

The Elect: yes, the Elect is a group of people (you could even call it a political movement)—capitalised the first time it is encountered and lowercased after that. Since there is nothing in the SEMOS about names of organisations and associations (please, correct me if I’m wrong!), I reached again for the CMOS: [8.65](https://archive.org/details/chicagomanualofs16edunse/page/414/mode/2up) tells us that political organisations should be capitalised.

I’ve again made the change as requested. Just like the previous one, though, if you think it should be reverted let me know and I’ll drop the commit.

Parts and volumes: parts were updated to volumes across the production. However, the fact that the book was split into three books (or volumes) is mostly something due to the publisher and the availability of paper—the logical divisions are more parts than volumes. Other productions semantically call parts what the printed page calls a book or volume, e.g., [this one](alain-rene-lesage_gil-blas_tobias-smollett) names the files books but uses the part semantic, and [this one](https://github.com/standardebooks/laurence-sterne_the-life-and-opinions-of-tristram-shandy-gentleman) names the files parts, uses the part semantic, but identifies them as books in the text.

**Comment by weijia-cheng** [comment]
Ok, I think there are two remaining things:

- [x] I originally misread the "Elect" commit. At first I thought that you had made it lowercase like "ex-queen" when I asked you to revert, but now I see that you were uppercasing it. (To avoid confusions like this, I like to title my commit message something like `[Editorial] Normalize name: elect ->  Elect` to make it clear what the direction of the change is on first glance). I agree with you that it should be normalized as "Elect" as that would be most readable and it is in line with CMOS. Sorry for making you go in a circle on this one!
- [x] `build-toc` creates changes, because currently it places the volume links after the individual chapters.

As for the volume thing, I just think that it's really confusing to have the text call it one thing and the semantic type another. The semantic types are so vaguely defined in the epub standard anyway that we might as well do whatever makes the code easiest to understand :)

**Comment by Gragt** [comment]
Yes, that sounds less ambiguous. I’ll do that in future. Several words in this book were spelled in as much as three different ways, so that made things interesting.

I updated the manifest and the spine but it looks like I forgot the TOC. Ah well. Again, good catch.

I guess that is all, then. Unless you find something else, feel free to close this issue if you are satisfied with the current state of things.

**Comment by weijia-cheng** [comment]
Yep, this looks good, thanks!
